README.md

```diff
@@ -4,26 +4,53 @@
 ![CI](https://github.com/actions/typescript-action/actions/workflows/ci.yml/badge.svg)
 [![Check dist/](https://github.com/actions/typescript-action/actions/workflows/check-dist.yml/badge.svg)](https://github.com/actions/typescript-action/actions/workflows/check-dist.yml)
 
-This action allows you to create an annotation in Axiom.
-
-## Usage
-
-1. Create an Axiom API token with permission to create annotations
-1. Create your annotation:
-
-```yaml
-- name: Create annotation
-  uses: axiomhq/annotation-action@v0.1.0
-  with:
-    axiomToken: ${{ secrets.AXIOM_TOKEN }}
-    datasets: production-logs
-    type: "production-release"
-    time: "2024-01-01T00:00:00Z"            # optional, defaults to now
-    endTime: "2024-01-01T01:00:00Z"         # optional, defaults to null
-    title: "Production release"             # optional
-    description: "Commit ${{ github.sha }}" # optional
-    url: "https://axiom.co"                 # optional, defaults to job url
-```
+This GitHub Action lets you create annotations in [Axiom](https://axiom.co/).
+For more information, see the
+[Axiom documentation](https://axiom.co/docs/query-data/annotate-charts).
+
+You can configure GitHub Actions using YAML syntax. For more information, see
+the
+[GitHub documentation](https://docs.github.com/en/actions/learn-github-actions/understanding-github-actions#create-an-example-workflow).
+
+## Prerequisites
+
+- [Create an Axiom account](https://app.axiom.co/).
+- [Create a dataset](https://axiom.co/docs/reference/datasets) with the name
+  `my-dataset`.
```

Comment:
> with the name `my-dataset`.

do we need this part? this works with any dataset not just ones with this name. 🤔